dashboard/src/api/TreeDetails.tsx

```diff
@@ -244,15 +244,18 @@ const fetchTreeCommitHistory = async (
   origin: string,
   gitUrl: string,
   gitBranch: string,
+  filters: TTreeDetailsFilter,
 ): Promise<TTreeCommitHistoryResponse> => {
+  const searchParam = mapFiltersToUrlSearchParams(filters);
```

Comment:
in the future we should remove this `mapFiltersToUrlSearchParams` so can you add a TODO to not use this? Axios can handle the serialization